compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -21,6 +21,20 @@ function parseStmt(Tokenizer tok) returns Stmt|err:Syntax {
     Position startPos = tok.currentStartPos();
     match cur {
         [IDENTIFIER, var identifier] => {
+            var peeked = tok.peek(true);
```

Comment:
With boolean arguments, it's clearer to use a named argument i.e `tok.peek(skipQualIdent=true);`

Reply:
We don't always do this, but I think we should aim too.